cmake/common.cmake

```diff
@@ -25,29 +25,6 @@ find_package(Threads REQUIRED)
 
 set(PYTHON unbuffer python3)
 
-set(SERVICE_IDENTITY_CURVE_CHOICE
```

Comment:
Is `DEFAULT_PARTICIPANTS_CURVE` still used in `function(add_e2e_test)`?

Reply:
It was, but I don't think it needed to be - have removed it (if we change the default value in `curve.h`, it may not match the default argument to `--participants-curve`. I think this is fine).